test/e2e/e2e_evictioninbackground_test.go

```diff
@@ -0,0 +1,542 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/apimachinery/pkg/util/wait"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	componentbaseconfig "k8s.io/component-base/config"
+	"k8s.io/klog/v2"
+	utilptr "k8s.io/utils/ptr"
+
+	kvcorev1 "kubevirt.io/api/core/v1"
+	generatedclient "kubevirt.io/client-go/generated/kubevirt/clientset/versioned"
+
+	"sigs.k8s.io/descheduler/pkg/api"
+	apiv1alpha2 "sigs.k8s.io/descheduler/pkg/api/v1alpha2"
+	"sigs.k8s.io/descheduler/pkg/descheduler/client"
+	"sigs.k8s.io/descheduler/pkg/framework/plugins/defaultevictor"
+	"sigs.k8s.io/descheduler/pkg/framework/plugins/podlifetime"
+)
+
+const (
+	vmiCount = 3
+)
+
+func virtualMachineInstance(idx int) *kvcorev1.VirtualMachineInstance {
+	return &kvcorev1.VirtualMachineInstance{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: fmt.Sprintf("kubevirtvmi-%v", idx),
+			Annotations: map[string]string{
+				"descheduler.alpha.kubernetes.io/request-evict-only": "",
```

Comment:
just to confirm, the feature depends only on the key, and not on the value? Would be good to add a little a bit of the documentation above `EvictionRequestAnnotationKey` and `EvictionInProgressAnnotationKey`

Reply:
That is correct. +1 for a documentation. I still have a blog post to write.

Reply:
Ok, sounds good. Let's not forget about the constants and update them after as well.